projects/netgrif-components-core/src/lib/panel/abstract-panel.component.ts

```diff
@@ -1,13 +1,16 @@
 import {AfterViewInit, Component, EventEmitter, Input, OnDestroy, Output, TemplateRef, ViewChild} from '@angular/core';
 import {MatExpansionPanel} from '@angular/material/expansion';
 import {CaseListFontColorService} from '../utility/service/case-list-font-color.service';
+import { ActivatedRoute, Params } from '@angular/router';
+import { config } from 'rxjs';
```

Comment:
what does this `config` import do? The only `config` in the file is a variable name not a type nor a method

Reply:
I think this was accidentally added. Removed.